src/Search/Api/SearchController.php

```diff
@@ -134,7 +136,11 @@ public function indexAction(Request $request, Accept $type)
                 break;
         }
 
-        $data = $this->client->searchDocuments($query->getRawQuery());
+        try {
+            $data = $this->client->searchDocuments($query->getRawQuery());
+        } catch (NoNodesAvailableException $e) {
```

Comment:
Does this wrap timeout errors from cURL?

Reply:
It is what comes out from the black box ElasticSearch SDK (see https://elifesciences.atlassian.net/browse/ELPP-3447?focusedCommentId=30254&page=com.atlassian.jira.plugin.system.issuetabpanels:comment-tabpanel#comment-30254) as it doesn't expose the underlying requests and implements some (unused so far) retry logic.

Reply:
We can handle things as we see them. Was trying to understand how it works a bit (looks to unfortunately use RingPHP+cURL).

Reply:
Uhm I wonder which of the twelve Guzzle major versions that corresponds to

Reply:
Part of the short-lived Guzzle 5.